ozaria/site/components/interactive/PageInteractive/draggableOrdering/index.vue

```diff
@@ -61,6 +72,59 @@
         }
 
         return undefined
+      },
+
+      userAnswer () {
+        return this.promptSlots.map((s) => s.id)
+      },
+
+      solutionCorrect () {
+        for (let i = 0; i < this.userAnswer.length; i++) {
+          if (this.userAnswer[i] !== this.localizedInteractiveConfig.solution[i]) {
+            return false
+          }
+        }
+
+        return true
+      }
+    },
+
+    methods: {
+      async submitSolution () {
+        this.showModal = true
+        this.submitEnabled = false
+
+        // TODO save through vuex and block progress until save is successful
+        // TODO fix submission format
```

Comment:
are these TODOs pending for now or for later?

Reply:
Same for all 3 types of interactives

Reply:
submission format has been fixed however these will still need Vuex integration at some point.